### PR TITLE
[CatalogPromotions] Rename CP Rule to Scope

### PR DESCRIPTION
**Pull request by arti0090**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

The Catalog Promotion Rule is now renamed to Scope - this naming was good in the case of Cart Promotion (which was the inspiration for this Rule) but it looks like the Rule is not an appropriate naming. Thus the decision to rename to scope 🚀

### CODE REVIEW

File: src/Sylius/Bundle/CoreBundle/Migrations/Version20211001073918.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20211001073918 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
```
Comment (AdamKasp) [inline src/Sylius/Bundle/CoreBundle/Migrations/Version20211001073918.php:17]:
```suggestion
        return ' Renames rule to scope in Catalog Promotion feature';
```
or sth similar

**Review by AdamKasp — APPROVED** [review]

File: src/Sylius/Bundle/PromotionBundle/Form/Type/CatalogPromotionType.php [hunk 2]
```diff
@@ -51,7 +51,7 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             ] )
             ->add('rules', CollectionType::class, [
```
Comment (AdamKasp) [inline src/Sylius/Bundle/PromotionBundle/Form/Type/CatalogPromotionType.php:52]:
```suggestion
            ->add('scopes', CollectionType::class, [
```

File: src/Sylius/Bundle/PromotionBundle/Form/Type/CatalogPromotionType.php [hunk 3]
```diff
@@ -51,7 +51,7 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
...
                 'label' => 'sylius.ui.rules',
```
Comment (AdamKasp) [inline src/Sylius/Bundle/PromotionBundle/Form/Type/CatalogPromotionType.php:53]:
```suggestion
                'label' => 'sylius.ui.scopes',
```

File: features/promotion/applying_catalog_promotions/reapplying_catalog_promotions_after_editing_their_rules.feature [hunk 4]
```diff
@@ -1,8 +1,8 @@
 @applying_catalog_promotions
```
Comment (GSadee) [inline features/promotion/applying_catalog_promotions/reapplying_catalog_promotions_after_editing_their_rules.feature:1]: You should also change the name of filename

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, @arti0090! :tada:

**Pull request merged**